arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+        <cuboid id="zone-2b" min="-11,1,44" max="-12,23,20"/>
+    </union>
+    <negative id="not-buildable">
+        <cuboid id="buildable" min="-10,0,-42" max="11,21,43"/>
+    </negative>
+    <cylinder id="original-block-protection" base=".5,0,.5" radius="100" height="176"/>
+    <!-- spawn locations -->
+    <point id="team-one-spawn-point">${team-one-spawn}</point>
+    <point id="team-two-spawn-point">${team-two-spawn}</point>
+    <union id="spawn-protections">
+        <cuboid id="team-one-spawn-protection" min="-3,20,-42.5" max="4,66,-36.5"/>
+        <cuboid id="team-two-spawn-protection" min="-3,20,43.5" max="4,66,36.5"/>
+    </union>
+    <union id="cages">
+        <cylinder id="team-one-spawn-cage" base="${team-one-spawn}" height="4" radius="4"/>
+        <cylinder id="team-two-spawn-cage" base="${team-two-spawn}" height="4" radius="4"/>
+    </union>
+    <!-- bed locations -->
+    <cuboid id="bed-red-1" min="0,10,-40" max="1,11,-41"/>
+    <cuboid id="bed-red-2" min="0,10,-41" max="1,11,-42"/>
+    <cuboid id="bed-blue-1" min="0,10,41" max="1,11,42"/>
+    <cuboid id="bed-blue-2" min="0,10,42" max="1,11,43"/>
+    <!-- applies -->
+    <apply enter="never" region="ceiling" message="You may not climb higher!"/>
+    <apply enter="never" region="no-go" message="You may not leave the course!"/>
+    <apply block="original-block-protection-filter" message="You can't modify that block!"/>
+    <apply block-place="never" region="not-buildable" message="You may not build in this area!"/>
+    <apply block-break="red-bed-protect-1" region="bed-red-1" message="You may not break your own bed!"/>
+    <apply block-break="red-bed-protect-2" region="bed-red-2" message="You may not break your own bed!"/>
+    <apply block-break="blue-bed-protect-1" region="bed-blue-1" message="You may not break your own bed!"/>
+    <apply block-break="blue-bed-protect-2" region="bed-blue-2" message="You may not break your own bed!"/>
+    <apply block-place="never" region="spawn-protections" message="You may not build in the spawn area!"/>
+    <apply kit="void-death" region="void"/>
+</regions>
+<portals sound="false" observers="never">
+    <portal destination="team-one-spawn-point" forward="should-tp" filter="only-team-one" yaw="@0" pitch="@0"/>
+    <portal destination="team-two-spawn-point" forward="should-tp" filter="only-team-two" yaw="@180" pitch="@0"/>
+</portals>
+<score>
+    <limit>${score-limit}</limit>
+</score>
+<itemremove>
+    <item>iron sword</item>
+    <item>iron pickaxe</item>
+    <item>iron axe</item>
+    <item>leather helmet</item>
+    <item>leather chestplate</item>
+    <item>leather leggings</item>
+    <item>leather boots</item>
+    <item>stained clay</item>
+    <item>golden apple</item>
+    <item>bed</item>
+    <item>sandstone</item>
+    <item>wood</item>
+</itemremove>
+<kill-rewards>
+    <kill-reward>
+        <item material="golden apple" amount="1"/>
```

Comment:
`amount="1"` is redundant here, you can take it out unless you plan on changing it in the future

Reply:
True, will update